src/transformers/searchservice/createSpellCheck.ts

```diff
@@ -5,5 +5,6 @@ export function createSpellCheck(data: any): SpellCheck {
     originalQuery: data.originalQuery,
     correctedQuery: data.correctedQuery.value,
     type: data.type,
+    matchedSubstrings: data.correctedQuery.matchedSubstrings
```

Comment:
can `data.correctedQuery.matchedSubstrings` be undefined?

Reply:
Looks like [product says](https://yext.slack.com/archives/C032CKFARGS/p1658842703108109) that a spell check object will always have `matchedSubstrings`!